mag2exp/fmr.py

```diff
@@ -0,0 +1,116 @@
+"""FMR submodule.
+
+Module for calculation of Ferromagnetic Resonance related
+quantities.
+"""
+
+from typing import Optional, Tuple
+
+import discretisedfield as df
+import hvplot.xarray  # noqa
+import micromagneticdata as micd
```

Comment:
Like in the notebook
```suggestion
import micromagneticdata as mdata
```